CMakeLists.txt

```diff
@@ -39,11 +39,26 @@ option(AVIF_CODEC_SVT "Use the SVT-AV1 codec for encoding" OFF)
 option(AVIF_CODEC_AOM_DECODE "if AVIF_CODEC_AOM is on, use/offer libaom's decoder" ON)
 option(AVIF_CODEC_AOM_ENCODE "if AVIF_CODEC_AOM is on, use/offer libaom's encoder" ON)
 
-option(AVIF_LOCAL_AOM "Build the AOM codec by providing your own copy of the repo in ext/aom (see Local Builds in README)" OFF)
-option(AVIF_LOCAL_DAV1D "Build the dav1d codec by providing your own copy of the repo in ext/dav1d (see Local Builds in README)" OFF)
-option(AVIF_LOCAL_LIBGAV1 "Build the libgav1 codec by providing your own copy of the repo in ext/libgav1 (see Local Builds in README)" OFF)
-option(AVIF_LOCAL_RAV1E "Build the rav1e codec by providing your own copy of the repo in ext/rav1e (see Local Builds in README)" OFF)
-option(AVIF_LOCAL_SVT "Build the SVT-AV1 codec by providing your own copy of the repo in ext/SVT-AV1 (see Local Builds in README)" OFF)
+option(AVIF_LOCAL_AOM "Build the AOM codec by providing your own copy of the repo (see Local Builds in README)" OFF)
+option(AVIF_LOCAL_DAV1D "Build the dav1d codec by providing your own copy of the repo (see Local Builds in README)" OFF)
+option(AVIF_LOCAL_LIBGAV1 "Build the libgav1 codec by providing your own copy of the repo (see Local Builds in README)" OFF)
+option(AVIF_LOCAL_RAV1E "Build the rav1e codec by providing your own copy of the repo (see Local Builds in README)" OFF)
+option(AVIF_LOCAL_SVT "Build the SVT-AV1 codec by providing your own copy of the repo (see Local Builds in README)" OFF)
+
+set(AVIF_LOCAL_AOM_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/ext/aom" CACHE PATH "Source directory of your own copy of AOM codec")
```

Comment:
These paths look quite random. If we instead 'install' these packages, they can be more consistent and maybe we can just set a prefix, and then search under `include` and `lib` dirs. But this means during development you'll need to 'install' instead of 'build' each time you make some changes to some codec.

Reply:
My bad, I forgot to explain the 'install' I said.
I'm quoting 'install' to mean we are not going to really copy the files into system directory, but to use install ability to copy necessary headers and library files into a standard path structure under build directory. This way they can be found by simply provide a prefix path, and this is exactly what we've done for rav1e.